content/doc/book/managing/cli.adoc

```diff
@@ -204,27 +206,81 @@ The general syntax for invoking the client is as follows:
 
 [source]
 ----
-java -jar jenkins-cli.jar [-s JENKINS_URL] command [options...] [arguments...]
+java -jar jenkins-cli.jar [-s JENKINS_URL] [global options...] command [command options...] [arguments...]
 ----
 
 The `JENKINS_URL` can be specified via the environment variable `$JENKINS_URL`.
+Summaries of other general options can be displayed by running the client with no arguments at all.
 
-[TIP]
-====
-The `JENKINS_URL` environment variable is automatically set when Jenkins forks a process
-during builds or Pipelines, allowing the use of the Jenkins CLI from inside a
-project without explicit configuration of the Jenkins URL.
-====
+=== Client connection modes
+
+There are three basic modes in which the 2.54+ client may be used, selectable by global option:
```

Comment:
(version number here and elsewhere to be adjusted according to actual merge target)